pages/profile/index.tsx

```diff
@@ -24,6 +25,14 @@ function ProfilePage(): ReactElement {
       if (user) {
         setProfile(user);
         setAuthenticatedType('private');
+        window.$chatwoot.setCustomAttributes({
```

Comment:
instrument.js:109 TypeError: Cannot read properties of undefined (reading 'setCustomAttributes')

<img width="1405" alt="Screen Shot 2022-01-04 at 3 00 15 PM" src="https://user-images.githubusercontent.com/5401575/148070494-f21fd3e9-93f1-4fc0-af14-498393b17060.png">

Reply:
fixed